introspect.go

```diff
@@ -53,18 +53,18 @@ func AccessTokenFromRequest(req *http.Request) string {
 }
 
 func (f *Fosite) IntrospectToken(ctx context.Context, token string, tokenType TokenType, session Session, scopes ...string) (TokenType, AccessRequester, error) {
-	var found bool = false
+	var found = false
 	var foundTokenType TokenType = ""
 
 	ar := NewAccessRequest(session)
 	for _, validator := range f.TokenIntrospectionHandlers {
 		tt, err := validator.IntrospectToken(ctx, token, tokenType, ar, scopes)
-		if err := errors.Cause(err); err == nil {
+		if err == nil {
```

Comment:
I simplified the logic here a bit. `ErrorToRFC6749Error` uses `Cause` internally as well, so I think this now is clearer and easier to read.